### PR TITLE
Add docs for installing with stack

### PR DESCRIPTION
**Pull request by jakequade**
Adds some general documentation for getting the exercises up and running with stack. Loved the YLJ19 workshop!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,30 @@ a course about testing, each level is a separate `test-suite` in the
...
+# initialise
+$ stack init
+
+# add dependencies
+$ stack add hedgehog
+$ stack add tasty
+$ stack add tasty-hedgehog
+
+# run ghci
+stack ghci
```
Comment (endgame) [inline README.md]: Some of the other stack-users said that `stack test --file-watch :level01` and `stack ghci :level01` worked. If this is true for your machine, can you please revise this? That way we won't have to ask users to run a bunch of package-management commands.

If you don't have time to revise this, can you please test the commands and let me know whether they ran or not? Then I can merge and update the docs.

**Review by endgame — COMMENTED** [review]
Good stuff, thank you.

File: README.md (hunk 1 above)
Comment (jakequade) [reply README.md]: `stack test --file-watch :level01` worked a treat - updated. Thanks for the feedback!

**Comment by endgame** [comment]
Sean tells me that there were some hard-to-debug weirdness with file-watch, so I'll merge this and fix it up after. Thanks.

**Comment by jakequade** [comment]
Thanks @endgame - appreciate the feedback. First open-source PR, hope it wasn't too shabby.

**Comment by endgame** [comment]
All good mate, all good.

**Pull request merged**